src/lib/net.js

```diff
@@ -681,3 +681,65 @@ foam.CLASS({
     }
   ]
 });
+
+
+foam.ENUM({
```

Comment:
TODO: We should early-bind environment. We can't load things for web in NodeJS in a 3rd party tool right now, anyway.